plugin/evm/vm.go

```diff
@@ -865,6 +865,16 @@ func (vm *VM) Shutdown(context.Context) error {
 	return nil
 }
 
+// newBlock returns a new Block wrapping the ethBlock type and implementing the snowman.Block interface
+func (vm *VM) newBlock(ethBlock *types.Block) (*Block, error) {
```

Comment:
@qdm12 I was going to change it to `newBlock(vm *VM, ethBlock *types.Block)` and then realized we actually have `vm.NewBlock` for extensible interface. So I think this is probably better to define in vm so we can keep `vm.NewBlock` and `vm.newBlock` aligned.